### DISCUSSION TITLE
Production Ready

### DISCUSSION THREAD

**Discussion by mgarf** [socketio/socket.io-deno#1] [Q&A] [answered]
Excited to see this take form. Definitely something that I will be looking at to migrate my existing socket io implementation on node. Can't wait for the redis adapter to migrated as that is my only blocker

Is this considered production ready?

**Comment by darrachequesne** [comment, marked as answer]
Hi! Thanks a lot for the feedback, that's really valuable :+1: 

Porting existing adapters like the Redis one is indeed at the top of our todo-list :angel: 

Regarding the production readiness, this implementation does pass the test suite [there](https://github.com/socketio/engine.io-protocol#test-suite) and the test coverage is already quite high, so I'm fairly confident (though bug-free sofwares are rare, of course).

**Reply by darrachequesne** [reply]
Update: the Redis adapter is now available!

Release notes: https://github.com/socketio/socket.io-deno/releases/tag/0.2.0

**Reply by pagoru** [reply]
Amazing job! 🚀
